CHANGELOG.md

```diff
@@ -7,15 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.5.0]
-### Uncategorized
-- Add beta and Flask builds to CI ([#12572](https://github.com/MetaMask/metamask-extension/pull/12572))
-- Check if ledger was successfully able to establish transport on confirm screen mount ([#12535](https://github.com/MetaMask/metamask-extension/pull/12535))
-- Ledger WebHID What's New popup ([#12501](https://github.com/MetaMask/metamask-extension/pull/12501))
-- Ensure ledger transport preference is set on import ([#12557](https://github.com/MetaMask/metamask-extension/pull/12557))
-- Don't show ledger live option in advanced settings if using firefox ([#12494](https://github.com/MetaMask/metamask-extension/pull/12494))
-- Turn off all old notifications ([#12500](https://github.com/MetaMask/metamask-extension/pull/12500))
-- Provide instructions for users switching from Ledger Live to WebHID ([#12519](https://github.com/MetaMask/metamask-extension/pull/12519))
-- Connect Ledger via WebHID ([#12411](https://github.com/MetaMask/metamask-extension/pull/12411))
+### Added
+- [#12411](https://github.com/MetaMask/metamask-extension/pull/12411): Add support for connecting Ledger devices to MetaMask via WebHID
+- [#12501](https://github.com/MetaMask/metamask-extension/pull/12501): Add "What's New" notification regarding Ledger WebHID support
+
+### Changed
```

Comment:
Nit: "Removed" might be more appropriate here

```suggestion
### Removed
```

Reply:
I can change this, I just noticed we haven't used the `### Removed` header before so `### Changed` seemed the best for log consistency

Reply:
updated